kuma/wiki/tests/test_content.py

```diff
@@ -994,6 +994,35 @@ def test_annotate_links_external_link():
     assert normalize_html(actual_raw) == expected
 
 
+@pytest.mark.parametrize(
+    "code",
+    (
+        # Simple existing class value
+        [
+            '<pre class="syntaxbox">h1 { color:brown }</pre>',
+            '<pre class="syntaxbox notranslate">h1 { color:brown }</pre>',
+        ],
+        # No class attribute at all
+        [
+            "<pre>h1 { color:pink }</pre>",
+            '<pre class="notranslate">h1 { color:pink }</pre>',
+        ],
+        # 'notranslate' already set
+        [
+            '<pre class="notranslate brush:js">h1 { color:brown }</pre>',
+            # Note, no difference!
+            '<pre class="notranslate brush:js">h1 { color:brown }</pre>',
+        ],
+    ),
```

Comment:
Nit: Would be good to add human-readable `ids` here for easy identification of the case in the `pytest` output on failure, something like:
```suggestion
    ),
    ids=("simple", "noclass", "nochange"),
```

Reply:
Thanks! Fixed.